src/pages/Profile/index.jsx

```diff
@@ -29,12 +37,23 @@ export default function Profile() {
       }
     };
   }
-  function isAvailableInTheList(list) {
-    return function (id) {
-      return list.find((game) => game.id === id);
-    };
+
+  function dragStartHandler(list, setList, data, ev) {
+    setDataTransfer({ from: [list, setList], data: { ...data } });
+    ev.dataTransfer.dropEffect = "link";
   }
 
+  function dropHandler(list, setList, event) {
+    event.preventDefault();
```

Comment:
The same goes here, you can pass status as an argument here, and based on the drop status and drag card status you know where to move the card.

Reply:
Fixed